code/modules/holiday/holidays.dm

```diff
@@ -314,6 +316,30 @@
 	begin_day = 14
 	begin_month = DECEMBER
 
+/datum/holiday/thanksgiving
+	name = "Thanksgiving"
+	begin_week = 4
+	begin_month = NOVEMBER
+	begin_weekday = THURSDAY
+
+/datum/holiday/columbus
+	name = "Columbus Day"
+	begin_week = 2
+	begin_month = OCTOBER
+	begin_weekday = MONDAY
+
+/datum/holiday/mother
+	name = "Mothers' Day"
+	begin_week = 2
+	begin_month = MAY
```

Comment:
@Really-Good-Soda-Flavor Actually, as Jordie has pointed out, don't include mother's day at all. (would probably make sense to remove father's day in tandem, too)